spectrum/src/main/java/io/github/giulong/spectrum/enums/LocatorType.java

```diff
@@ -0,0 +1,12 @@
+package io.github.giulong.spectrum.enums;
+
+public enum LocatorType {
```

Comment:
I did all the refactoring of this class, in conclusion it has been deleted because the logic is now managed by the LocatorType Enum. 

I do not resolve the conversation for you to check and leave me a feedback 👍 

commit 5665e0173786a045bd39b12619d029e5890561d1